integration_tests/jasmine_async/__tests__/async_test_fails.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @jest-environment jsdom
```

Comment:
Right now there is no way to assert what `testEnvironment` a test is run in. But I doubt someone will change this anyways, so shouldn't be a problem